### DISCUSSION TITLE
Who mixed GM_kit_demo_1.ogg?

### DISCUSSION THREAD

**Discussion by bobesponja93** [hydrogen-music/hydrogen#1942] [General]
This file was not simply exported from Hydrogen. It was mixed using other tools. Same for the other ogg files in the **demos** folder at [Sourceforge](https://sourceforge.net/projects/hydrogen/files/Hydrogen/demos/)

I would like to ask the person who mixed GM_kit_demo_1.ogg how he or she got the GM kit crash sample to sound the way it does at 00:13

**Comment by theGreatWhiteShark** [comment]
> I would like to ask the person who mixed GM_kit_demo_1.ogg how he or she got the GM kit crash sample to sound the way it does at 00:13

Uh. That thing is ancient. The version uploaded to SourceForge is more than 20 years old. That might have been done by Comix - the original author of Hydrogen.

But you should also note that the "GM kit" used for the snippet is _not_ the "GMRockKit" we ship along Hydrogen these days. The first one was the former default kit but had to be replaced due to licensing issues with the included samples. But I do not know when this exchange took place. It was way before I got involved.

**Comment by bobesponja93** [comment]
Alright, thanks for pointing that out.
So I guess what I'd like to see is finished Hydrogen songs, either on their
own or as part of an arrangement of several instruments.

Is there a place where people share those? are there known producers that
make their drum parts with Hydrogen?

**Reply by theGreatWhiteShark** [reply]
Well, you can find some songs in here https://github.com/hydrogen-music/Song-and-pattern-repository and there are a couple of people using Hydrogen to actively create music on https://linuxmusicians.com/

You may notice that these aren't too much songs. This is because in Hydrogen pattern - let alone songs - are bound to a certain drumkit. In most cases you can not switch kits without adjusting all patterns. This is more or less a show stopper for exchanging songs/pattern like we do for drumkits. But that might change with the 1.3 version of Hydrogen.
